RecoParticleFlow/PFClusterProducer/plugins/LocalMaximumSeedFinder.h

```diff
@@ -19,7 +20,10 @@ class LocalMaximumSeedFinder final : public SeedFinderBase {
   const int _nNeighbours;
 
   const std::unordered_map<std::string,int> _layerMap;
-  std::array<std::pair<double,double>, 35> _thresholds;
+
+  typedef std::tuple<std::vector<int> ,std::vector<double> , std::vector<double> > _i3tuple;
```

Comment:
start local variables and data members with lower case letter. Capitalization is to be used for class and type names.